Documentation/devel/release.md

```diff
@@ -71,13 +71,15 @@ Now we switch to the GitHub web UI to conduct the release:
 
 - Publish the release!
 
+- Clean your git tree: `sudo git clean -ffdx`.
+
 Now it's announcement time: send an email to rkt-dev@googlegroups.com describing the release.
 Generally this is higher level overview outlining some of the major features, not a copy-paste of the release notes.
 Use your discretion and see [previous release emails](https://groups.google.com/forum/#!forum/rkt-dev) for examples.
 Make sure to include a list of authors that contributed since the previous release - something like the following might be handy:
 
 ```
-	git log ...v0.16.0 --pretty=format:"%an" | sort | uniq | tr '\n' ',' | sed -e 's#,#, #g' -e 's#, $#\n#'
+	git log ...v1.0.0 --pretty=format:"%an" | sort | uniq | tr '\n' ',' | sed -e 's#,#, #g' -e 's#, $#\n#'
```

Comment:
We want the commits from `v1.0.0` to `v1.1.0`, and we get those with `HEAD...v1.0.0 == ...v1.0.0`, oder?